livechat/overview/get_started.rst

```diff
@@ -0,0 +1,83 @@
+==========================
+Get Started with Live Chat
+==========================
+
+Live chat is a cost-effective tool and a great way to bring visitors back to your website. It allows
+fast responses, human interaction, sales improvement, accessible and convenient support, and even
+services costs reduction. Your customers want to talk to you, so let's make it easy.
```

Comment:
I still think this doesn't work. You're approaching it in the wrong way, it's not positive enough. 

So I would approach with Live chat is an amazing tool that can help your customers in many ways. 
It has the highest satisfaction rating of any communication, it allows your customers to keep doing what they are doing while talking to your employees... 

You can talk about  cost effective but it should be the last part. 

The last sentence is good! 

There's many ways to sell live chat that is better than this. 

ideas:https://acquire.io/blog/live-chat-winning-customer-support-race/ 
https://neilpatel.com/blog/live-chat-marketing/